v2.1/reference/configuration/tidb-server/configuration-file.md

```diff
@@ -3,6 +3,8 @@ title: TiDB 配置文件描述
 category: reference
 ---
 
+<!-- markdownlint-disable MD001 -->
+
 # TiDB 配置文件描述
 
 <!-- markdownlint-disable MD001 -->
```

Comment:
We need to keep this line together with line#6?

Reply:
> We need to keep this line together with line#6?

No, it's duplicating the line 10. We should remove the line 10.